djangochannelsrestframework/extras/managers.py

```diff
@@ -0,0 +1,57 @@
+from typing import Any, Iterable, List, Optional, Sequence
+from django.db.models import Manager
+from djangochannelsrestframework.extras import (
+    post_bulk_create,
+    pre_bulk_create,
+    post_bulk_update,
+    pre_bulk_update,
+)
+
+
+class BulkManager(Manager):
+    """Bulk manager for sending bulk signals."""
+
+    def bulk_create(self, objs: Iterable[Any], **kwargs) -> List[Any]:
+        """Bulk create overide
+
+        Warning:
+            This method **DOESN'T RETURN** the `AutoIncrement` fields, this means, that the ID is `None`
+            in the instance.
+
+        Notes:
+            The ID is currently returned only in `PostgresSQL` or `MariaDB 10.5+`.
+        """
+        pre_bulk_create.send(
+            sender=objs[0].__class__, instances=objs, created=True, **kwargs
+        )
+        items = super().bulk_create(objs, **kwargs)
+        post_bulk_create.send(
+            sender=objs[0].__class__, instances=objs, created=True, **kwargs
```

Comment:
this will crash if you call bulk create with an emty list, best just to check and return before doing anything.

Reply:
Good catch